dp/buy_sell_stock.py

```diff
@@ -23,12 +23,13 @@
 def max_profit(prices):
     """
     input: [7, 1, 5, 3, 6, 4]
-    diff : [X,-6, 5,-2, 3,-2]
+    diff : [X,-6, 4,-2, 3,-2]
```

Comment:
@ankit167 
 I just revised the code according to your review.
May you review it again if your are free? Or just edit or delete this PR is if doesn't meet the standard.

The original author just  wrote this way in comment (without space before minus numbers), so I follow him.